ui/cockpit/client/scripts/camunda-cockpit-bootstrap.js

```diff
@@ -15,6 +15,11 @@
  * limitations under the License.
  */
 
+/*eslint-disable */
+window._import = path => {
+  return import(path);
```

Comment:
```suggestion
  // eslint-disable-next-line
  return import(path);
```
Can we disable eslint for this line and change the ecma version back to `2020`?